java/ql/src/semmle/code/java/security/XmlParsers.qll

```diff
@@ -545,10 +545,12 @@ class SafeSAXParser extends MethodAccess {
   }
 }
 
-/* SAXReader: https://cheatsheetseries.owasp.org/cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.html#saxreader
```

Comment:
The autoformat output below is garbled because the first commit erroneously removed the closing `*/` from the above comment. If you (remove the autoformat commit and) re-insert the missing `*/`, then the file is correctly auto-formatted.

Reply:
Ah, indeed. I should have paid a bit more attention to the changes I was making...